docs/learn/learn-async-backing.md

```diff
@@ -34,43 +34,51 @@ slashed.
 In synchronous backing, parablock generation is tightly coupled to the relay chain's progression on
 a one-to-one basis. Every parablock must be generated and backed within a relay-chain block
 (six-second window), and (if successfully backed) it will be included in a relay-chain block (often
-referred to as the **relay parent**, or latest block on the relay chain, as the parablock anchors itself to it) after an additional six
+referred to as the **relay parent** as the parablock anchors itself to it) after an additional six
 seconds. Thus, a parablock can be produced every 12 seconds because a new parablock can be produced
-after including the previous one.
+after including the previous one. Note [candidate receipts](#candidate-receipt) and not the
+parablocks themselves are included in relay parents.
 
-The **contextual execution** of new parablocks are parablock ancestors included in the relay chain
-after 12 seconds, where contextual execution refers to the context built by included parablock
-ancestors used by the parachains to generate new parablocks.
+### Sync Backing Mechanics
 
-:::info Candidate Recepts
+The diagram below shows the pipelining table for synchronous backing.
 
-- Not the entire parablock, but the **candidate receipt** consisting of the hash of the parablock,
-  state roots and ID info is placed on the parent block on the relay chain.
-- The relay chain does not access the entire state of a parachain but only the values that changed
-  during that block and the merkelized hashes of the unchanged values.
+![sync-backing-pipeline](../assets/sync-backing-pipeline.png)
 
-:::
+Backing (B) and inclusion (I) of each parablock happen on different relay chain blocks, and new
+parablocks are generated once the previous parablock has been included.
+[Contextual execution](#contextual-execution) is driven by the latest included parablock in the most
+recent relay parent. After 24 seconds, two parablocks, P1 and P2, have been included in the relay
+chain, and P3 has been backed.
 
-### Synchronous Backing Mechanics
+In synchronous backing we can imagine parablocks on a conveyor belt. The belt is synched with the
+relay chain: It takes six seconds (one relay block) to generate and back a parablock. It then takes
+another six seconds to make that parablock available and mark it as included on the relay parent.
```

Comment:
The relay parent of a parablock and the relay block including that parablock are always different.
```suggestion
another six seconds to make that parablock available and mark it as included in a relay block.
```